### PR TITLE
Clean up and remove unnecessary properties from gemspec

### PR DESCRIPTION
**Pull request by jrolfs**
- Remove now extraneous `executables` and `test_files` properties from spec
- Consistent comments and quoting 😬

### CODE REVIEW

File: brainstemjs.gemspec [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
-  gem.homepage      = "http://github.com/mavenlink/brainstem-js"
+  gem.authors       = ['Mavenlink']
+  gem.email         = ['opensource@mavenlink.com']
+  gem.description   = 'The Brainstem API adapter library for Backbone.js'
+  gem.summary       = 'Easily connect Backbone.js with Brainstem APIs.  Get relational models in Backbone.'
+  gem.homepage      = 'http://github.com/mavenlink/brainstem-js'
 
   gem.files         = `git ls-files`.split($/)
-  gem.executables   = gem.files.grep(%r{^bin/}).map{ |f| File.basename(f) }
-  gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
```
Comment (naganowl) [inline brainstemjs.gemspec]: I'm assuming these two lines aren't needed?

**Comment by naganowl** [comment]
Looks good.

**Comment by diclophis** [comment]
@jrolfs is this still a thing?

**Comment by diclophis** [comment]
closing, please re-open if desired

**Comment by jrolfs** [comment]
This should be mergeable

**Comment by jrolfs** [comment]
@diclophis

**Comment by juanca** [comment]
Thanks for the follow up @jrolfs

**Pull request merged**